README.md

```diff
@@ -258,6 +273,7 @@ Arguments:
 - name: The name of the publication
 - collection: A function that returns the collection to publish (or just a collection, if it's insecure).
 - selector: (Optional) A function that returns mongo selector that will limit the results published (or just the selector).
+- settings: (Optional) A object with settings on server side's publish function. (Details see below)
```

Comment:
Shouldn't this be a client-side option? getFilterQuery also gets called on the client, and you could pass the option to the server-side publish from there.